kafka-connect/kafka-connect-events/src/main/java/org/apache/iceberg/connect/events/CommitCompletePayload.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iceberg.connect.events;
+
+import java.util.UUID;
+import org.apache.avro.Schema;
+import org.apache.avro.SchemaBuilder;
+import org.apache.iceberg.avro.AvroSchemaUtil;
+
+/**
+ * A control event payload for events sent by a coordinator that indicates it has completed a commit
+ * cycle. Events with this payload are not consumed by the sink, they are informational and can be
+ * used by consumers to trigger downstream processes.
+ */
+public class CommitCompletePayload implements Payload {
+
+  private UUID commitId;
+  private Long validThroughTs;
+  private final Schema avroSchema;
```

Comment:
Should we include any additional information in this (like a List of `TableName`s that were committed?).  I think for this to be useful for downstream consumers, it has surprisingly little info.  This might also be helpful because not all tables may get commits, right?

Other information that could be useful (if available) would be stats (e.g. how may records were added.  How many tables were updated).  We can always add more to this, so think maybe the list of tables would be a good starting point.

Reply:
There is a separate message that is sent for each individual table commit, this message is sent after all of the individual table commit messages have been sent, as a sort of terminator.

Reply:
Never mind, it looks like that info is part of `CommitTablePayload`.